rts/Game/UI/MouseHandler.cpp

```diff
@@ -679,10 +686,9 @@ void CMouseHandler::WarpMouse(int x, int y)
 		return;
 
 	lastx = x + globalRendering->viewPosX;
-	lasty = y + globalRendering->viewPosY;
+	lasty = y;
```

Comment:
Values received in this function are already translated to internal mouse coordinates.